target/linux/ath79/image/generic-tp-link.mk

```diff
@@ -191,6 +191,34 @@ define Device/tplink_archer-d50-v1
 endef
 TARGET_DEVICES += tplink_archer-d50-v1
 
+define Device/tplink_archer-d7-v1
+  $(Device/tplink-qca-dsl)
+  ATH_SOC := qca9558
+  DEVICE_MODEL := Archer D7
+  DEVICE_VARIANT := v1
+  DEVICE_PACKAGES := kmod-usb2 kmod-usb-ledtrig-usbport kmod-ath10k-ct ath10k-firmware-qca988x-ct
+  IMAGE_SIZE := 15936k
+  TPLINK_HWID := 0x89300001
+  TPLINK_HWREV := 0x0000002D
+  TPLINK_FLASHLAYOUT := 16Mqca
+  TPLINK_HWREVADD := 0x00000002
+endef
+TARGET_DEVICES += tplink_archer-d7-v1
+
+define Device/tplink_archer-d7b-v1
+  $(Device/tplink-qca-dsl)
+  ATH_SOC := qca9558
```

Comment:
Please rename this to "SOC" due to 95030ac0b673165772daf27128359eaaf536d457.

Reply:
Fixed!